docs/sources/developers/plugins/publishing-and-signing-criteria.md

```diff
@@ -6,26 +6,37 @@ title: Plugin publishing and signing criteria
 
 Grafana plugins must adhere to the following criteria when being reviewed for publishing and signing.
 
+> **Important:** Grafana Labs reserves the right to decline or remove any plugin at its discretion. Failure to comply with publishing and signing criteria may result in immediate removal from the Grafana plugin catalog.
+
 ## Privacy and security
 
```

Comment:
Adding the section titles make this much easier to scan. 

Should you have a sentence between headings?

Reply:
I added a sentence between the two headings. Will check in soon.

Reply:
Checked in fix, available for review.